source/getting-started/index.rst

```diff
@@ -1,131 +1,411 @@
+reserve your resources before you can launch an instance on them. Chameleon
+supports both *on-demand* and *advanced* reservations. We will use an on-demand
+reservation for this guide, but note that you can reserve resources in advance,
+which is often necessary to get access to popular, scarce hardware like GPUs.
+
+Step 1: Access a Testbed Site
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As mentioned above, different Chameleon sites have different hardware. To log
+in to a Chameleon site from the main Chameleon_ portal page, click on the
+"Experiment" tab on the nav bar at the top. From the dropdown, select a
+Chameleon site. We will be working with nodes available in either |CHI@UC|
+or |CHI@TACC|, so you can select one of those.
+
+.. figure:: ../_static/imgs/getting_started/experiment-dropdown.png
+   :align: center
+   :figwidth: 50 %
+   :figclass: screenshot
+
+   Select a site to use.
+
+When you access one of the sites, you are first taken to a site dashboard,
+which shows a summary of your project's current resource usage. The dashboard
+looks something like this:
 
 .. figure:: dashboard.png
    :alt: The Chameleon Dashboard's resource usage summary
    :figclass: screenshot
 
    An overview of your project's current resource usage
 
-Reserving a node
-----------------
+Once on your dashboard on the site, notice that the URL has changed to a
+specific domain for the testbed site we chose. You can also see which site you
+are currently on by clicking on the dropdown next to the Chameleon logo at the
+top left of the window.
+
+.. figure:: ../_static/imgs/getting_started/change-site-project-menu.png
+   :figwidth: 80 %
+   :align: center
+
+This section tells you which project you are currently using and which site. By
+clicking on the dropdown menu, you can change to another Chameleon site or
+change to another project.
+
+.. important::
+   Projects will only appear as an option in this menu if they have a current
+   active allocation of compute resources.
+
+.. figure:: ../_static/imgs/getting_started/leases-side-bar.png
+   :figwidth: 20 %
+   :align: left
+
+Step 2: Go to the Reservations Dashboard
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We need to reserve a ``Cascadelake R`` node for our use. From the main page of
+our testbed site, we can select the "Reservations" menu item on the side nav
+bar and then click "Leases." Doing so will open a new page showing a table of
+your lease activity. If you are a first-time user of Chameleon, the table will
+be blank. Let's change that now!
+
+Step 3: Creating a New Lease
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Before we create a lease, it is a good idea to check the :ref:`Host Calendars
+<the-lease-calendars>` available on the Leases page to see if another user has
+the hardware reserved.
+
+.. figure:: ../_static/imgs/getting_started/host-calendar-button.png
+
+Click the "Host Calendar" button now. You will be taken to a new page with a
+Gnatt chart.
```

Comment:
Gnatt -> Gantt

I'm not sure how many people are familiar with this term, maybe we should call this a timeline?